test/test_pte.cpp

```diff
@@ -229,7 +229,11 @@ auto TestPTE(const std::string name, const std::size_t nscratch_vars,
   }
   std::cout << std::endl;
 
+#ifdef PORTABILITY_STRATEGY_KOKKOS
+  return rho_v;
+#else
   return rho_d;
+#endif // PORTABILITY_STRATEGY_KOKKOS
```

Comment:
When Kokkos is on, this was actually not valid, because `rho_d` points to memory managed by Kokkos. This resolves that issue.